api/errors.go

```diff
@@ -0,0 +1,74 @@
+package api
+
+const (
+	// IPFSConnectionError is an error used for ipfs connection failures
+	IPFSConnectionError = "failed to connect to ipfs"
```

Comment:
this shouldn't be in the diff anymore right, now that #177 has been merged? 🤔

Reply:
Might've been because of how I rebased before-hand? I just did another rebase so it should be good now.